config/env/staging/plugins.js

```diff
@@ -24,8 +24,8 @@ module.exports = ({ env }) => ({
       // secureConnection: false,
       // tls: { ciphers: 'SSLv3'},
       auth: {
-        user: env('SMTP_USER'),
-        pass: env('SMTP_PASSWORD'),
+        user: 'sis.admin@medha.org.in',
+        pass: 'Admin.SIS407',
```

Comment:
these needs to come from env. This is an open-source repository. Pushing passwords to the code can lead to big issues!